merger/merger.go

```diff
@@ -248,3 +245,23 @@ func Match(rules []*rule.Rule, x *rule.Rule, info rule.KindInfo) (*rule.Rule, er
 
 	return nil, nil
 }
+
+func AttrMatch(x, y *rule.Rule, key string) bool {
```

Comment:
Let's leave this un-exported for now. Seems likely to change.

Reply:
Current merger_test.go is an external test. I have to create another internal test file to contain `TestAttrMatch`. How do I update `internal/go_repository_tools_srcs.bzl` after creating a new file?

Reply:
Actually, could you add cases to `TestMatch` instead of creating `TestAttrMatch`? Better to test against the exported API.

If you need to add a file in the future, you can `go run internal/list_repository_tools_srcs.go -generate internal/go_repository_tools_srcs.bzl`.